### PR TITLE
Week 2 corey cass

### PR DESCRIPTION
**Pull request by TheLavitz**
# Submitting Week 2 Assessment

### ✅ Completed Files

Select the following as confirmation that the files in this repository are complete.

- [X] Career Readiness
- [X] Coding Challenges
- [X] Coding Critique
- [X] Coding Concepts
- [X] Interview Questions

### ✋ Reminder

Do NOT merge your assessment!

### 📝 Feedback

If you would like specific feedback please leave details here:

### CODE REVIEW

File: career-readiness.md [hunk 1]
```diff
@@ -8,17 +8,17 @@ This week, we covered the importance of a great technical portfolio to help you
 
 - Spend time looking at other developer’s portfolios. Submit the links to three portfolios you will use for inspiration.
 
-1.
-2.
-3.
+1. https://www.seanhalpin.xyz/
+2. https://caferati.me/portfolio
+3. https://denisechandler.com/
```
Comment (chrisyhlee) [inline career-readiness.md:13]: Wow, these are awesome!

File: code-critique.js [hunk 2]
```diff
@@ -6,5 +6,118 @@
...
+//             return;
+//         }
+//     }
+//     console.log("No vowels found in the string");
+// }
+
+// Critique the code output.
+
+// This is a very interesting question, because as you see you can get wildly different answers depending on which GPT you ask and how your session is configured. Of the two JavaScript answers that Bing provided the "Precise" setting seems to have provided slightly more robust code but failed to include any code invoking the function to give an output and determine if the code work. Also included is an extra "return" that just returns "undefined" when the function is invoked. The "Creative" Bing instance provided JavaScript as well, which is functional, does include code invoking the function, and is commented to explain what each line of code is doing but in both cases the AI decided to go with loops and conditionals instead of a higher-order-function like .map(), which I would definitely attempt. I would say I've learned not to take code generated by AI at face value and to certainly continue to look for better answers even if the code that was generated does function as intended.
```
Comment (chrisyhlee) [inline code-critique.js:121]: Love that you tested many different AI tools, and fantastic critique.

**Review by chrisyhlee — APPROVED** [review]
This is a great assessment, well done! This clearly displays Competent. My only suggestion for this week is to elaborate a little more on interview questions. A tip to ensure the interviewer has no doubts on if you know what you're talking about or not is to add in practical examples of the concepts in question, or scenarios where you have used the concept yourself. Great work, I hope to continue seeing this level in the future weeks!

**Comment by TheLavitz** [comment]
> This is a great assessment, well done! This clearly displays Competent. My only suggestion for this week is to elaborate a little more on interview questions. A tip to ensure the interviewer has no doubts on if you know what you're talking about or not is to add in practical examples of the concepts in question, or scenarios where you have used the concept yourself. Great work, I hope to continue seeing this level in the future weeks!

I went back and forth for a while on whether or not to include practical examples, so that's good to know.

**Pull request merged**